server/test/controllers/applicant/ApplicantRequestedActionTest.java

```diff
@@ -0,0 +1,62 @@
+package controllers.applicant;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.Test;
+
+public class ApplicantRequestedActionTest {
```

Comment:
Do we need a test for if there are query params or an anchor tag as part of the URL?

Reply:
Good idea, updated!